lib/utils.py

```diff
@@ -100,6 +105,61 @@ def backslash_escape_layer(argv: List[str]) -> None:
         argv[i] = re.sub(r"\\(.)", "\\1", argv[i])
 
 
+def set_some_extras(
```

Comment:
Small comment.  Is this ONLY needed for DAGs?  If so, could the function name be changed to `set_dag_extras` or something along those lines?

Reply:
On second thought - this seems to be required for DAGs, but for regular jobs, other settings are needed.  If so, ignore my last comment.

Reply:
Right, I'm just partitioning it to a subset of things we need in condor_submit_dag, then there's a longer list for jobsub_submit, where we fix disk units, etc.   Maybe "some" is a poor choice... sigh.